tools/secrets-tool/commandhelper.cpp

```diff
@@ -65,6 +65,39 @@ static Sailfish::Crypto::CryptoManager::DigestFunction digestEnum(const QString
     return Sailfish::Crypto::CryptoManager::DigestSha512;
 }
 
+static QVariantMap toCustomParameters(const QStringList &options)
+{
+    QVariantMap customs;
+    for (QList<QString>::ConstIterator it = options.constBegin(); it != options.constEnd(); it++) {
+        QStringList keyValue(it->split('='));
```

Comment:
instead of using split, could instead use split+join, to allow options like '-o someKey="someValueWith=Sign"' (that is, just split out the LHS of the first = as the key, and join the rest to form the value.